src/main/java/songbox/house/service/UserService.java

```diff
@@ -7,7 +7,8 @@
 
 public interface UserService {
 
-    UserInfo findByUserName(final String userName);
+    UserInfo findByUserName(String userName);
+    UserInfo findByUserNameOrCreate(String userName);
```

Comment:
What's the purpose of this method? We can't create a user without password

Reply:
I'll remove it